pom.xml

```diff
@@ -84,8 +84,8 @@
     </dependency>
     <dependency>
       <groupId>org.apache.commons</groupId>
-      <artifactId>commons-lang3</artifactId>
-      <version>${org.apache.commons-lang3.version}</version>
+      <artifactId>commons-text</artifactId>
```

Comment:
@wbrunette Why was this changed to `commons-text`? Is it used in `odk-rest-interface`?

Reply:
yes it's used in odk-rest-interface

Reply:
specifically it's used for html escaping